packages/flutter_tools/lib/src/commands/attach.dart

```diff
@@ -315,8 +316,12 @@ class AttachCommand extends FlutterCommand {
         result = await runner.attach();
         assert(result != null);
       }
-      if (result != 0)
+      if (result == 0) {
```

Comment:
Why not report success/failure for every command here:
https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/runner/flutter_command.dart#L442 ?

Reply:
That's good idea, we can filter down the attach success specifically from that, but it will give us other good information too.

cc @blasten

Reply:
sg. I filed https://github.com/flutter/flutter/issues/33779